Doc/library/profile.rst

```diff
@@ -123,13 +123,15 @@ them in various ways.
 The file :mod:`cProfile` can also be invoked as a script to profile another
 script.  For example::
 
-   python -m cProfile [-o output_file] [-s sort_order] myscript.py
+   python -m cProfile [-o output_file] [-s sort_order] [-m module] myscript.py
```

Comment:
Note that `-m module` and `myscript.py` are exclusive. So the doc could/should be spelled `(-m module | myscript.py)`.